src/jemalloc_cpp.cpp

```diff
@@ -224,27 +241,48 @@ alignedSizedDeleteImpl(void* ptr, std::size_t size, std::align_val_t alignment)
 	if (unlikely(ptr == nullptr)) {
 		return;
 	}
+	LOG("core.alignedSizedDeleteImpl.entry", "ptr: %p, size: %zu, flags: %d", ptr,
```

Comment:
alignedSizedDeleteImpl isn't a public API and is used by `operator delete(void* ptr, std::size_t size, std::align_val_t alignment)`. Maybe:

`LOG("core.opeartor_delete.entry", "ptr: %p, size: %zu, alignment: %zu", ptr, size, alignment);`

Since we are making changes here, let's also improve a few other places:
1) Remove the LOG here:
https://github.com/jemalloc/jemalloc/blob/e4817c8d89a2a413e835c4adeab5c5c4412f9235/include/jemalloc/internal/jemalloc_internal_inlines_c.h#L581
Then add LOG into sizedDeleteImpl above, which matches the other operator delete LOGs but with a size param.

2) Similarly, move the one in imalloc_fastpath to its callers, so it can differentiate malloc vs operator new: https://github.com/jemalloc/jemalloc/blob/dev/include/jemalloc/internal/jemalloc_internal_inlines_c.h#L309

Do we need to include jemalloc/internal/log.h here?

Reply:
Sounds good, to confirm, the idea here is to log the highest level public api, is that correct? I will also double check other places if any.

> Do we need to include jemalloc/internal/log.h here?

That is already implicitly included through `internal_includes.h`.

Reply:
> the idea here is to log the highest level public api, is that correct?

Right. That allows the log info to be better matched with the user side callsites, e.g. sized aligned delete should not log the MALLOCX flag (which is the internal impl to that API), but the input param of size and align. While the sdallocx itself logs the flag because that's what the users pass in.